src/main/resources/templates/utility/tillsession.html

```diff
@@ -35,6 +35,8 @@
                     </tr>
                 </table>
                 <br>
+                <div class="alert alert-info"><span class="badge badge-primary">NOTE:</span> Closing the till will print a report to: <span class="badge badge-info" th:text="${reportPrinterName}"></span></div>
```

Comment:
I like this message. I suggest adding a th:if= to test for `reportPrinterName == null or reportPrinterName.isBlank()`, and if it is null warn that the till report will not print automatically.

Reply:
Added to the PR